XenCert/StorageHandler.py

```diff
@@ -2781,103 +2800,274 @@ def MetaDataTests(self):
         Print("MetaDataTests not applicable to NFS SR type.")
         return False
 
+    def PoolTests(self):
```

Comment:
We could get rid of these pool-tests but I've got lots of complaints about these tests failing. 
My intention here was to create a shared NFS SR and ensure that it was accessible for all hosts within the pool. 

I could get rid of this entire section if necessary!